reverse default behavior for xattrs (#1788)
hpc/charliecloud: doc/ch-image.rst

    which risks corruption but may be OK for some workloads.

  :code:`--no-xattrs`

>     Enforce default handling of xattrs, i.e. do not save them in the build cache
>     or restore them on rebuild. Used to override the :code:`CH_XATTRS` environment
>     variable.

Reviewer: Possibly?
```suggestion
    Enforce default handling of xattrs, i.e. do not save them in the build cache
    or restore them on rebuild. This is the default, but the option is provided
    to override the :code:`$CH_XATTRS` environment variable.
```

named :code:`.git*` or other Git metadata are permitted in the image’s root
directory.

`Extended attributes <https://man7.org/linux/man-pages/man7/xattr.7.html>`_
(xattrs) are ignored by the build cache by default. Cache support for xattrs

> belonging to unprivileged namespaces (e.g. :code:`user`) can be enabled by

Reviewer: "in unprivileged xattr namespaces" to clarify we're not talking about container namespaces?

> specifying the :code:`--xattrs` option, or by setting the :code:`CH_XATTRS`

Reviewer: ```suggestion
specifying the :code:`--xattrs` option or by setting the :code:`CH_XATTRS`
```

> environment variable. If :code:`CH_XATTRS` is set, the user can manually
> override it via the command line by specifying the :code:`--no-xattrs` option,
> which will restore the default behavior for that command only. Note that
> extended attributes in privileged namespaces (e.g. :code:`trusted`) cannot be
> read by :code:`ch-image` and will always be lost without warning.

Reviewer: possibly?

```suggestion
environment variable. If :code:`CH_XATTRS` is set, you
override it with :code:`--no-xattrs`. **Note that
extended attributes in privileged xattr namespaces (e.g. :code:`trusted`) cannot be
read by :code:`ch-image` and will always be lost without warning.**
```
